hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -148,6 +170,31 @@ public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
     }
   }
 
+  private String getNodeId() {
```

Comment:
I don't think we need to complex logic for the name of the backup directory. Would something simpler suffice?